src/app/cup/cup-rating-group-table/cup-rating-group-table.component.html

```diff
@@ -0,0 +1,52 @@
+<div *ngIf="cupRatingGroup">
+   <table class="d-none d-lg-table table table-striped table-bordered shadow mb-0">
```

Comment:
looks like you forgot to copy tables for other screens:
`<table class="d-none d-sm-table d-lg-none table table-striped table-bordered shadow mb-0">`
and 
`<table class="d-table d-sm-none table table-sm table-striped table-bordered shadow mb-0">`